sources/us/pa/butler.json

```diff
@@ -15,29 +15,28 @@
             {
                 "name": "county",
                 "website": "https://www.butlercountypa.gov/353/Mapping",
+                "license": { "url": "https://www.butlercountypa.gov/353/Mapping" },
                 "contact": {
-                    "name": "DONNA ARMAHIZER",
-                    "title": "Director of Mapping",
-                    "phone": "724-284-5330",
+                    "name": "Ashley Leslie",
+                    "title": "Director of GIS & Mapping",
+                    "phone": "724-284-5329",
                     "email": "mapping@co.butler.pa.us",
                     "address": "124 West Diamond Street, Butler, PA 16001"
                 },
-                "data": "https://gis.co.butler.pa.us/arcgis/rest/services/PAT/ButlerPA_PAT_GIS/MapServer/0",
+                "data": "https://gis.co.butler.pa.us/arcgis/rest/services/PAT/ButlerPA_PAT_GIS/MapServer/1",
                 "protocol": "ESRI",
                 "conform": {
+                    "accuracy": 1,
                     "format": "geojson",
-                    "id": "PIN",
-                    "number": "ADDRESS",
-                    "street": [
-                        "PRD",
```

Comment:
This is ok, but I prefer to concatenate street name parts into a full name instead of using a full name so that if we ever wanted to pass the street name parts through to the output data it would be easier to make that change.